ChemoTracker/components/Profile/ProfileTop.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { Text, View, Image } from 'react-native';
+import { TabNavigator } from 'react-navigation';
+import styles from '../../styles/main.js';
+import color from '../../styles/color.js';
+
+class ProfileTop extends Component {
+	render() {
+		return (
+      <View style = {styles.profileTopContainer}>
+        <Image source = { require('../../res/profile-background.jpg') } style = {styles.profileBackground}>
```

Comment:
the image path should also not be hard coded but passed in as props, as per my comment on `ProfileDetailsTable` so you can just do `require(this.props.imagePath)` or something like that